07.ls_object/test/display_test.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'pathname'
+require_relative '../lib/display'
+
+class DisplayTest < Minitest::Test
+  TARGET_PATHNAME = Pathname('test/fixtures/sample_directory/*')
+
+  def test_render
+    paths = Dir.glob(TARGET_PATHNAME)
+    display = Display.new(paths)
+    expected = ['Gemfile            config             storage',
+                'Gemfile.lock       config.ru          test',
+                'README.md          db                 tmp',
+                'Rakefile           lib                vendor',
+                'app                log',
+                'bin                public']
+
+    assert_equal expected, display.render
+  end
+
+  def test_render_at_reverse
+    paths = Dir.glob(TARGET_PATHNAME).reverse
+    display = Display.new(paths)
+    expected = ['vendor             lib                Rakefile',
+                'tmp                db                 README.md',
+                'test               config.ru          Gemfile.lock',
+                'storage            config             Gemfile',
+                'public             bin',
+                'log                app']
+
+    assert_equal expected, display.render
```

Comment:
```suggestion
    expected = <<~TEXT.chomp
      vendor             lib                Rakefile
      tmp                db                 README.md
      test               config.ru          Gemfile.lock
      storage            config             Gemfile
      public             bin
      log                app
    TEXT

    assert_equal expected, display.render
```

renderは配列ではなく文字列を返すようにした方がテストが書きやすいかも？

Reply:
配列と文字列のどちらにするか迷って、「再利用しやすいかな？」という考えから配列にしました。
テストのしやすさは確かに文字列のほうが良いなぁと感じます😊
この辺はおそらく実際のデータの内容や再利用の可能性の高さ低さで変わってきそうですね。